scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -604,6 +605,8 @@ object TreeSyntax {
           case head :: Nil => s("${ ", head, " }")
           case other => s("${", r(other.map(i(_)), ""), n("}"))
         }
+      case Term.SplicedMacroPat(pat) =>
+        s("${ ", pat, " }")
```

Comment:
do we need spaces around `pat`? why not `"${"` and `"}"`?

Reply:
I copied line 605. I think if I change one I should change the other?

Reply:
sorry, didn't notice this. then let's keep it consistent with line 605 (as well as 595 and probably others). please ignore the original comment :)